test/specs/eazework.spec.ts

```diff
@@ -0,0 +1,21 @@
+import eazeworkPage from "../pageobjects/eazework.page.js";
+import credentails from "../data/crdentails.json" assert {type:"json"};
+
+describe('Feature: Leave Request Save in Draft', () => {
+    beforeEach(async () => {
+        await browser.url("https://www.eazework.com/Login/");
```

Comment:
@GautamCredex : Please make a new method in login.page.ts for login to the eazework dashboard. As going further we have to use the login method in different test suites. Also, make sure to have .env file and put all the credential there.